packages/scandipwa/src/route/MenuPage/MenuPage.container.js

```diff
@@ -19,8 +19,7 @@ import Menu from 'Component/Menu';
 import { updateMeta } from 'Store/Meta/Meta.action';
 import { changeNavigationState } from 'Store/Navigation/Navigation.action';
 import { TOP_NAVIGATION_TYPE } from 'Store/Navigation/Navigation.reducer';
-import { HistoryType } from 'Type/Router.type';
-import { appendWithStoreCode } from 'Util/Url';
+import history from 'Util/History';
```

Comment:
Since issue was actually in appendWithStoreCode can we get back history from router ?